### DISCUSSION TITLE
Shadow & Light and Enhanced (Again)

### DISCUSSION THREAD

**Discussion by Koshi-v-dushe** [Shadow-and-Light/shadow-and-light#53] [Q&A] [answered]
Why don't these two plugins work together?
ElvUI Enhanced (Again) have the ultra-feature, which I want use or see in Shadow & Light - "Minimap button bar". It places all minimap addon's icons in the one frame.

Will this feature have implemented in Shadow & Light?

**Comment by Repooc** [comment, marked as answer]
> Why don't these two plugins work together? ElvUI Enhanced (Again) have the ultra-feature, which I want use or see in Shadow & Light - "Minimap button bar". It places all minimap addon's icons in the one frame.
> 
> Will this feature have implemented in Shadow & Light?

No it won't be added as u can find a similar feature in a standalone addon and in projectazilroka addon for the same feature

**Reply by Koshi-v-dushe** [reply]
Thanks for your answer! The only one addon I have found in Curse for this feature was bugged "Minimap Button Frame" (both of them), but ProjectAzilroka works even much better!

Thanks again!)

**Comment by Repooc** [comment]
No problem
